mathbot/modules/latex/__init__.py

```diff
@@ -236,6 +236,10 @@ def process_latex(latex, is_inline):
 	blockformat = re.match(BLOCKFORMAT_REGEX, latex)
 	if blockformat:
 		latex = blockformat[1].strip(' \n')
+	if SPOILER_REGEXP.search(latex):
+		latex = latex[3:-3]
+		print(latex)
```

Comment:
Is the `print(latex)` a leftover debug statement?

Reply:
yes, oops.